[ Backported.  ]

commit 6448aace637843e8e7c021d7f8c5d5d5fdd71974
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Wed Mar 3 20:19:48 2010 +0000

    Mention improvement for X86 general purpose registers.
    
    2010-03-03  H.J. Lu  <hongjiu.lu@intel.com>
    	    Eli Zaretskii  <eliz@gnu.org>
    
    	* NEWS: Add X86 general purpose registers section.

--- gdb-7.1-p3/gdb/NEWS	2010-03-18 22:01:55.000000000 +0100
+++ gdb-7.1/gdb/NEWS	2010-04-03 21:07:46.000000000 +0200
@@ -438,6 +438,14 @@ GDB will now correctly handle all of:
     now support hardware watchpoints, and will use them automatically
     as appropriate.
 
+* X86 general purpose registers
+
+  GDB now supports reading/writing byte, word and double-word x86
+  general purpose registers directly.  This means you can use, say,
+  $ah or $ax to refer, respectively, to the byte register AH and
+  16-bit word register AX that are actually portions of the 32-bit
+  register EAX or 64-bit register RAX.
+
 * Python scripting
 
   GDB now has support for scripting using Python.  Whether this is
